src/bsoncxx/test/v1/array/view_string_maker.cpp

```diff
@@ -0,0 +1,59 @@
+// Copyright 2009-present MongoDB, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// This file is included by mongocxx/test/v1/bsoncxx.cpp!
```

Comment:
Remove or revise comment? `view_string_maker.cpp` is not `#include`ed but rather added to the source list.

```suggestion
```